src/pages/cli/geo/geofencing.mdx

```diff
@@ -0,0 +1,283 @@
+export const meta = {
+  title: `Geofencing (Developer Preview)`,
+  description: `Use Amplify CLI to create and manage collections of Geofences`,
+};
+
+Amplify's `geo` category enables you to create and manage collections of Geofences used to setup virtual geographic perimeters.
+
+<Callout>
+
+**Note:** The Amplify Geo Geofencing feature is in developer preview and is not intended to be used in production environments. Please reach out to us for any feedback and/or issues [here](https://github.com/aws-amplify/amplify-cli/issues)
+
+</Callout>
+
+## Install Amplify CLI
+Since the Geofencing feature is a developer preview, you need to update your Amplify CLI to try it out:
+
+```bash
+npm i -g @aws-amplify/cli@geo
+```
```

Comment:
Probably should add another callout or note to check the amplify cli version and it to be greater than a certain version

Reply:
Added below